code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -339,9 +339,6 @@
 		if (!M.anchored && !M.pulledby && M.last_high_pressure_movement_air_cycle < SSair.times_fired)
 			M.experience_pressure_difference(pressure_difference, pressure_direction)
 
-/atom/movable/var/pressure_resistance = 10
-/atom/movable/var/last_high_pressure_movement_air_cycle = 0
-
```

Comment:
I'm pretty sure this was here in an attempt to keep /atmospherics modular to some extent, similar to the turf vars at the top of the file. Should it stay with uh, less absolute pathing?